stamp_ldflags.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
Rename to `go_wrapped` or some other `go_*` ?

Reply:
Let's do `go_with_version.sh`.